waku/waku_api/rest/doc/storeapi.yaml

```diff
@@ -0,0 +1,138 @@
+# /store/v1/messages:
+get:
+  summary: Gets message history
+  description: >
+    Retrieves WakuV2 message history. The returned history
+    can be potentially filtered by optional request parameters.
+  operationId: getMessageHistory
+  tags:
+    - store
+  parameters:
+    - name: peerAddr
```

Comment:
Would it be good to define peerAddr as a type, as this maybe requried to be used in other API's as well.

Reply:
Agree, I will incorporate.